### PR TITLE
Pr/show nested subcomponents in expanded representation

### PR DESCRIPTION
**Pull request by dj-mcg**

### CODE REVIEW

**Comment by dj-mcg** [comment]
Hey folks, please let me know if I need to resubmit this PR, thanks so much!

**Comment by dj-mcg** [comment]
Hi, it looks like some remote builds are failing to launch in pre-flight for some reason. Is this something I need to resolve on my end?

**Comment by williamkrick** [comment]
> Hi, it looks like some remote builds are failing to launch in pre-flight for some reason. Is this something I need to resolve on my end?

No, there is an issue with the preflight system yesterday. I believe it is fixed now so I launched another run.

File: plugin/pxr/maya/lib/usdMaya/readJob_ImportWithProxies.cpp [hunk 1]
```diff
@@ -178,16 +178,23 @@ bool UsdMaya_ReadJobWithSceneAssembly::_ProcessProxyPrims(
...
         MStatus                  status;
         UsdMayaPrimReaderContext ctx(&mNewNodeRegistry);
 
-        // Get the geom root proxy shape node.
-        SdfPath proxyShapePath = pxrGeomRoot.GetPath().AppendChild(_tokens->MayaProxyShapeNodeName);
-        MObject proxyShapeObj = ctx.GetMayaNode(proxyShapePath, false);
+        // Get the exclude ancestor's proxy shape node.
+        SdfPath excludeAncestorPath = entry.first;
+        SdfPath proxyShapePath = excludeAncestorPath.AppendChild(
+            TfToken(TfStringPrintf("%sProxy", excludeAncestorPath.GetName().c_str())));
```
Comment (williamkrick) [inline plugin/pxr/maya/lib/usdMaya/readJob_ImportWithProxies.cpp:188]: Is there a reason "Proxy" is not in the local _tokens?

**Review by williamkrick — APPROVED** [review]

File: plugin/pxr/maya/lib/usdMaya/readJob_ImportWithProxies.cpp (hunk 1 above)
Comment (dj-mcg) [reply plugin/pxr/maya/lib/usdMaya/readJob_ImportWithProxies.cpp:188]: No, I can add it if you prefer!

**Comment by dj-mcg** [comment]
OK I tokenized that "Proxy" suffix, please have another look @williamkrick - thanks!

**Review by williamkrick — APPROVED** [review]

**Pull request merged**